hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -693,10 +694,13 @@ public void testPutKey()
 
     for (int i = 0; i < 10; i++) {
       String keyName = UUID.randomUUID().toString();
+      Map<String, String> metadata = new HashMap<>(1);
+      metadata.put(OzoneConsts.GDPR_FLAG, Boolean.toString(i % 2 == 0));
+      Map<String, String> originalMetadata = ImmutableMap.copyOf(metadata);
 
       OzoneOutputStream out = bucket.createKey(keyName,
           value.getBytes().length, STAND_ALONE,
-          ONE, new HashMap<>());
+          ONE, metadata);
```

Comment:
Since this test is called PutKey, it is for generic putkey. To test the GDPR feature, we have dedicated tests in the same class. 
I recommend undoing this change.

Reply:
Done, thanks for the recommendation.